frontend-react/e2e/fixtures/idletimeout.spec.ts

```diff
@@ -0,0 +1,30 @@
+import process from "node:process";
+
+import { test, expect } from "@playwright/test";
+
+const timeout = parseInt(process.env.VITE_IDLE_TIMEOUT ?? "60000");
```

Comment:
We probably want to change these defaults of `60000` since they're exceed the test timeout and the test will fail.